src/components/regions/MunicipalityListBox.tsx

```diff
@@ -0,0 +1,49 @@
+import { Text } from "@/components/ui/text";
+import { LocalizedLink } from "@/components/LocalizedLink";
+import { useTranslation } from "react-i18next";
+import { SectionWithHelp } from "@/data-guide/SectionWithHelp";
+import { DataGuideItemId } from "@/data-guide/items";
+
+interface MunicipalityListBoxProps {
```

Comment:
Idea for future: It could be interesting to have a map of just the specific region with it's municipalities and their total emissions for comparison. Would be visually fun and a similar/familiar view to the rankings page.